lwgsm/src/include/lwgsm/lwgsm_ssl.h

```diff
@@ -0,0 +1,42 @@
+/**
+ * \file            lwgsm_ssl.h
+ * \brief           SSL API
+ */
+
+/*
+ * Copyright (c) 2020 Tilen MAJERLE
+ *
+ * Permission is hereby granted, free of charge, to any person
+ * obtaining a copy of this software and associated documentation
+ * files (the "Software"), to deal in the Software without restriction,
+ * including without limitation the rights to use, copy, modify, merge,
+ * publish, distribute, sublicense, and/or sell copies of the Software,
+ * and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+ * OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE
+ * AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+ * WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ * OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * This file is part of LwGSM - Lightweight GSM-AT library.
+ *
+ * Authors:          Tilen MAJERLE <tilen@majerle.eu>,
+ *                   Ilya Kargapolov <d3vil.st@gmail.com>
+ * Version:         v0.1.0
+*/
+
```

Comment:
No need for this space

Reply:
> 
> 
> > > Hi, looks good to me.
> > > Prior merging, I would kindly ask you to update syntax to be in-line with previous work.
> > 
> > 
> > Hi, thanks!
> > Actually I found some differences in MQTT AT commands between SIM800 and SIM7000, could you point for example how to handle this differences?
> 
> For the moment such thing is not available :) Can we first focus on one device and then once it works, we try to implement second?
> 
> Since you have added many things (big thanks to this), I would kindly ask you to:
> 
>     * Update files to be in line with code syntax style, especially on tabs and spaces
> 
>     * Update header files to support doxygen generation
> 
>     * Add modules to docs folder for sphinx docs (I think you will get it?)
> 
> 
> Let me know if too complex for you.

Sure, could you please point me to the tool which could check code-style?
I'm using simple editor which not aware about that.
Let you know once done.